docs/example-scenario/hpc/video-rendering.md

```diff
@@ -0,0 +1,187 @@
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+
+[Learn more about ARM templates][azure-arm-templates]
+
+### Components
+
+Azure Batch builds upon the following Azure technologies:
+
+* [Resource Groups][resource-groups] is a logical container for Azure resources.
+* [Virtual Networks][vnet] are used to for both the Head Node and Compute resources
+* [Storage][storage] accounts are used for the synchronisation and data retention
+* [Virtual Machine Scale Sets][vmss] are utilised by CycleCloud for compute resources
+
+## Machine Sizes available for Azure Batch
+While most rendering customers whill choose resources with high CPU power, other workloads using VM Scale Sets may choose VM's differently and will depend on a number of factors:
+  - Is the Application being run memory bound?
+  - Does the Application need to use GPU's? 
+  - Are the job types embarassingly parallel or require Infiniband connectivity for tightly coupled jobs?
+  - Require fast I/O to Storage on the Compute Nodes
+
+Azure has a wide range of VM sizes that can address each and every one of the above application requirements, some are specific to HPC, but even the smallest sizes can be utilised to provide an effective grid implementation:
+
+  - [HPC VM sizes][compute-hpc] Due to the CPU bound nature of rendering, Microsoft typically suggests the Azure H-Series VM's.  These are built specifically available for high end computational needs, have 8 and 16 core vCPU sizes available, and feature DDR4 memory, SSD temporary storage and Haswell E5 Intel technology.
+  - [GPU VM sizes][compute-gpu] GPU optimized VM sizes are specialized virtual machines available with single or multiple NVIDIA GPUs. These sizes are designed for compute-intensive, graphics-intensive, and visualization workloads.
+    - NC, NCv2, NCv3, and ND sizes are optimized for compute-intensive and network-intensive applications and algorithms, including CUDA and OpenCL-based applications and simulations, AI, and Deep Learning. NV sizes are optimized and designed for remote visualization, streaming, gaming, encoding, and VDI scenarios utilizing frameworks such as OpenGL and DirectX.
+  - [Memory optmised VM sizes][compute-memory] When more memory is required, the memory optimized VM sizes offer a higher memory-to-CPU ratio.
+  - [General purposes VM sizes][compute-general] General purpose VM sizes are also available and provide balanced CPU-to-memory ratio.
+
+
+### Alternatives
+
+If you require more control over your rendering environment in Azure or need a hybrid implementation, then CycleCloud computing can help orchestrate an IaaS grid in the cloud. Using the same underlying Azure technologies as Azure Batch, it makes building and maintaining an IaaS grid an efficient process. To find out more and learn about the design principles, please use the following link:
```

Comment:
There doesn't seem to be a following link, also it should read something like:

Review [design principles][design-principles] to learn how to...

and then add [design-principles]: at the bottom with the appropriate link.